.github/workflows/test-suite-web-e2e.yml

```diff
@@ -13,6 +13,7 @@ on:
     paths-ignore:
       - "suite-native/**"
       - "packages/connect*/**"
+      - "packages/suite-desktop*/**"
```

Comment:
btw this doesn't seem to be working, is the syntax OK? it did trigger suite-web e2e. cc @vdovhanych